src/interface.jl

```diff
@@ -10,6 +10,9 @@ set_current(o::Buffer) = set_current_buffer(o.client, o)
 set_current(o::Window) = set_current_window(o.client, o)
 set_current(o::Tabpage) = set_current_tabpage(o.client, o)
 
+cursor(c::Window) = get_cursor(c) + [0, 1]
```

Comment:
Good points. I'm refactoring buffer indexing right now to get around some edge cases with end=-1, but once that's done, I'll implement those suggestions. Thanks for the feedback!